api/.pipeline/config.js

```diff
@@ -111,6 +112,7 @@ const phases = {
     appHost: staticUrls.test,
     backboneApiHost: 'https://api-test-biohub-platform.apps.silver.devops.gov.bc.ca',
     backboneIntakePath: '/api/dwc/submission/intake',
```

Comment:
Could you update this submission path to be `.../queue` as well, and the one below on line 115.